### DISCUSSION TITLE
terminal command to export config

### DISCUSSION THREAD

**Discussion by randallwc** [rxhanson/RectanglePro-Community#287] [General]
is there a terminal command to export my config as json?

I have found the analogous of this for hookshot but this doesn't store in plaintext.
https://github.com/rxhanson/Rectangle#preferences-storage

**Comment by rxhanson** [comment]
What is your use case? There isn't a json export via terminal at the moment, but could see adding it in if the use-case is valuable enough to be used by a lot of people.

Indeed, you could use the user defaults (com.knollsoft.Hookshot), just like Rectangle, and that's what I was going to suggest. Most of the plist should be human-readable. It would also be good to know what you are looking to modify via text editor here that requires human readability.

**Reply by randallwc** [reply]
my use-case is that i want to track my config changes in plain text json format via a github repo, and i want to save preferences via a script.
